src/main/java/com/training/sample/pages/Assignment5.java

```diff
@@ -0,0 +1,13 @@
+package com.training.sample.pages;
+
+public class Assignment5 {
+    public int getStringIndex(String[] names, String word) {
```

Comment:
lets also add a logic in this method where it throws a java exception when there is no match. 
Also add a test for the same.